### DISCUSSION TITLE
Font thicken for Linux?

### DISCUSSION THREAD

**Discussion by ndavd** [ghostty-org/ghostty#4067] [Feature Requests, Ideas] [answered]
Hi, would it be possible in the future to have font thicken like it is setup in Kitty?

I'm on Linux and this is my configuration for Kitty and how the font looks:
```
font_family JetBrainsMono NFM
font_size 10
macos_thicken_font 0.75
```
![image](https://github.com/user-attachments/assets/0fb9cfc1-09af-41a3-b57c-7117cfd9eda1)
While on Ghostty:
```
font-family = JetBrainsMono Nerd Font
font-size = 10
font-thicken = true
```
![image](https://github.com/user-attachments/assets/3e167ca0-baec-481d-a049-4c5fb2f791b9)

**Comment by 00-kat** [comment, marked as answer]
FYI (from Qwerasd on Discord):
> `font-thicken` adjusts how we ask CoreText to render the glyphs from the font, it's just a different anti-aliasing mode which results in a thicker appearance

**Comment by ad-on-is** [comment]
you can try using ` font-style = semibold # (or bold)`
